client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/fields.js

```diff
@@ -65,7 +65,8 @@ const AddressFields = ( props ) => {
 
 	const fieldErrors = isObject( errors ) ? errors : {};
 	const getId = ( fieldName ) => group + '_' + fieldName;
-	const getValue = ( fieldName ) => values[ fieldName ] || '';
+	const unescapeHtml = ( value ) => unescape( value ).replace( /&#039;/g, "'" );
```

Comment:
Eeek. We're going to need a more comprehensive solution, no?

Reply:
Maybe `decodeEntities` from [lib/formatting](https://github.com/Automattic/wp-calypso/blob/master/client/lib/formatting/index.js#L13-L20)?